heroku/cli
# refactor(pg-v5): Move command pg:settings:log-statement to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bD1SzYAK/view

This converts `pg:settings:log-statement` to oclif and updates its associated tests.

### Testing

1) Pull down this branch
2) Run `./bin/run pg:settings:log-statement --app APP DB VALUE` with an app that has a postgres db where value is an a valid option ('none', 'ddl', 'mod', 'all')
3) Verify the command succeeds
4)  Run `./bin/run pg:settings:log-statement  --app APP DB` with the same app and see that it displays the changed value
5) Compare with prod
